kolena/_experimental/object_detection/utils.py

```diff
@@ -79,18 +94,29 @@ def _compute_threshold_curve(
             zero_division=0,
         )
 
-        precisions.append(precision)
-        recalls.append(recall)
+        # avoid curves with one x-value and two y-values
```

Comment:
I'm curious to know why this is necessary.  Wouldn't the plot still look the same?

Reply:
NVM I saw your comment above.